go/channelling/roomworker.go

```diff
@@ -146,6 +147,10 @@ func (r *roomWorker) Users() []*roomUser {
 	return users
 }
 
+func (r *roomWorker) GetType() string {
+	return r.Type
```

Comment:
r.Type should probably be unexported as there is a getter to avoid confusion / give hint that no direct access is intended.